manuscript/the-meat.txt

```diff
@@ -513,21 +513,21 @@ But the `getPreviousYearFragment` function is more complex:
 
         return fragment;
     },
+**
```

Comment:
Shit. That was a "fix" on my end to prevent the in-line code from fucking with my MD syntax highlighting. I  thought I went through and removed them. Sorry.